### PR TITLE
Docs - Add ShooterGame as a Reference in the Porting Guide

### PR DESCRIPTION
**Pull request by ernopp**
#### Description
Adds a callout suggesting ShooterGame as a reference project to consult while porting.

(the green box below)

2019-04-11 17_53_17-Window

#### Release note
* Added the ported ShooterGame as a reference project in the Porting Guide.

#### Tests
Tested with Improbadoc

### CODE REVIEW

File: SpatialGDK/Documentation/content/get-started/porting-unreal-project-to-gdk.md [hunk 1]
```diff
@@ -18,9 +17,17 @@ Before porting your project:
...
+* `<GameRoot>` - The directory containing your project's `.uproject` file and `Source` directory.  
+* `<ProjectRoot>` - The directory containing your `<GameRoot>`.  
+* `<YourProject>` - The name of your project's `.uproject` file (for example, `\<GameRoot>\TP_SpatialGDK.uproject`).
+
+<%(Callout type="tip" message="
+**ShooterGame Reference Project:** 
+
+As you port your own project, you may find it helpful to use our ported [Unreal ShooterGame](https://docs.unrealengine.com/en-us/Resources/SampleGames/ShooterGame) running on the GDK as a reference.
+
+You should already have this project set up as it is included in the  `Samples` directory of [the GDK's Unreal Engine fork](https://github.com/improbableio/UnrealEngine). [Here](https://www.youtube.com/watch?v=xojgH7hJgQs&feature=youtu.be) is a video showing what you can expect to see when running it.
```
Comment (m-samiec) [inline SpatialGDK/Documentation/content/get-started/porting-unreal-project-to-gdk.md]:
```suggestion
You should already have this project set up as it is included in the `Samples` directory of [the GDK's Unreal Engine fork](https://github.com/improbableio/UnrealEngine). [Here](https://www.youtube.com/watch?v=xojgH7hJgQs&feature=youtu.be) is a video showing what you can expect to see when running it. 
```

File: SpatialGDK/Documentation/content/get-started/porting-unreal-project-to-gdk.md [hunk 2]
```diff
@@ -18,9 +17,17 @@ Before porting your project:
...
-`<ProjectRoot>` - The directory containing your `<GameRoot>`.  
-`<YourProject>` - The name of your project's `.uproject` file (for example, `\<GameRoot>\TP_SpatialGDK.uproject`).
+* `<GameRoot>` - The directory containing your project's `.uproject` file and `Source` directory.  
+* `<ProjectRoot>` - The directory containing your `<GameRoot>`.  
+* `<YourProject>` - The name of your project's `.uproject` file (for example, `\<GameRoot>\TP_SpatialGDK.uproject`).
+
+<%(Callout type="tip" message="
+**ShooterGame Reference Project:** 
+
+As you port your own project, you may find it helpful to use our ported [Unreal ShooterGame](https://docs.unrealengine.com/en-us/Resources/SampleGames/ShooterGame) running on the GDK as a reference.
```
Comment (ElleEss) [inline SpatialGDK/Documentation/content/get-started/porting-unreal-project-to-gdk.md]:
```suggestion
As you port your own Unreal project to SpatialOS, you could use our pre-ported Unreal Shooter Game as a reference.
```
"running on" suggests it is already running (as in running right now). Also, if we are saying it is pre-ported, we don't need to say it runs on the GDK, I think?
Also, we probs don't want to link to it here as we say below that you probs already have it. So we send people off to get it and then say "Oh, by the way, you already have it, probs. LOL! Trolled you!"

File: SpatialGDK/Documentation/content/get-started/porting-unreal-project-to-gdk.md [hunk 3]
```diff
@@ -18,9 +17,17 @@ Before porting your project:
...
+
+You should already have this project set up as it is included in the `Samples` directory of [the GDK's Unreal Engine fork](https://github.com/improbableio/UnrealEngine). [Here](https://www.youtube.com/watch?v=xojgH7hJgQs&feature=youtu.be) is a video showing what you can expect to see when running it.
```
Comment (ElleEss) [inline SpatialGDK/Documentation/content/get-started/porting-unreal-project-to-gdk.md]:
```suggestion
You should already have this project as it is included in the `Samples` directory of [the SpatialOS Unreal Engine fork](https://github.com/improbableio/UnrealEngine) which you downloaded as part of the _Get started_ steps. <br/>
(If you don't have it, you can get it from the Unreal repository, and if you want to see the game running, there's a [video on youtube](https://www.youtube.com/watch?v=xojgH7hJgQs&feature=youtu.be) to check out.)
```
Slight re-jig to incorporate the link to the GDK and make it clear where and when you would have got it.

**Comment by ernopp** [comment]
Incorporated Laura feedback

2019-04-15 20_18_36-Window

**Pull request merged**